### DISCUSSION TITLE
Possible to order from other PCB houses?

### DISCUSSION THREAD

**Discussion by mwedmark** [frntc/SIDKick-pico#3] [General]
Hi! Thanks for sharing this ground-breaking SID-project. I've fiddled with Pico for VGA/HDMI uses just for fun and I see the potential here when it comes to audio signals. Sound great in the demos too! I would like to try this out and I order mostly from JLCPCB (where I have an order going for some other cool projects) or is this project sponsored by PCBWAY. Is there any way to order this board using JLCPCB? I can't seem to find the Gerber or the Kicad project, only the BOM and CPL. Thanks!

**Comment by frntc** [comment]
Hi,

I can of course understand your question. Let me explain why I decided to not release the Gerbers or KiCAD project for the SKpico yet. The reason is the (as I find) frustrating experiences with my previous projects: there are people who do not care about licenses, and produce and sell the hardware overpriced and for profit without even asking. (I’m not talking about spare PCBs that are left from a small batch, that would be fine!). As a consequence, I chose to release the hardware part, at least for now, via PCBWay -- everyone can order there (also with SMD parts prebuild) for reasonable prices without me giving everything out of my hands. For those who do not want to order there: restore-store.de will offer SKpicos soon for fair prices.

**Reply by mwedmark** [reply]
Thanks for your clear answer! I can understand your concern and you have every right to choose licenses. 

Making both the PCB AND PCB+components available is great! Very simple for anyone to order! I'm gonna check out a 5-card order to see if shipping and taxes are worth it to Sweden.

To comply with your license/price cap, I kept away from checking out/ordering/producing: SidKick, SideKick and RAD boards/cards even though I've been specifically asked to produce them. But a lot of others are building them and proabably skipping the "details". Reason being that, as a small hobby company in Sweden, it is not possible to sell these in small series with similar pricing as mentioned on the web. As an example, I've built and sold a bunch high-quality Kung Fu Flash cards with happy customers that are willing to pay for "locally" produced, legally-binded to guarantee and also support from me. I've also tried to kept reporting errors and good ideas back to the open-source project. I didn't have the time to actully get any of my pull requests approved yet because of lack of time mostly. Even if that didn't end up as cash in Kim's pocket, I hope I did some good for the project and spread the love. :-)

I don't think there will be a lot of millionaires in this small market!

My guess would be that, even if released fully, the market would compete with itself and in the end, everybody is a winner. Early adaptors will overpay and those who wait for a while will just choose the better price.

This is just my own thoughts. Please keep producing and sharing these projects just like you do!! There are lots of people that love your stuff, especially the Sidekick I've heard a lot of good stuff about.

**Reply by frntc** [reply]
Thanks for your understanding, there are clearly arguments for either way. Maybe I've had particularly disappointing experiences with a few individuals ;-)

Since I'm obviously not into this for making money (I'd better spent my time for something else then...) I'm always happy to figure out ways if someone wants to offer my projects to the community (overall, I'm happy if people find them interesting and get hands on them). Not difficult to find out my mail address :-)

**Reply by wthorch42** [reply]
I fully understand your position.  I wasn't actually asking for myself, but for someone else who asked me.  I've already placed my order through PCBWay.

I've never used JLCPCB to produce PCBs for me.  So I don't know much about their services.  Do they offer any similar service to what you used with PCBWay where you could upload your project to them as well and let people order through their service as well?  If that's possible, it would address your concerns and still allow people to use their preferred fabricator.  If not, no big deal.  PCBWay does a great job.

**Comment by frntc** [comment]
Unfortunately JLCPCB doesn't have a similar click-to-order service (I asked them, but not sure if it's even planned). There is a price difference between JLCPCB and PCBWay, but I think it's within an acceptable range (for small batches).

**Reply by mwedmark** [reply]
I ordered a lot of stuff from JLCPCB. With a few screenshots for suitable default choice of services, GERBER+BOM+POS files it all works perfect. There are plugins for Kicad to automatically create these files but of course you need to do some additional work like selecting components from LCSC selection for Assembly to work. But I was very pleased with the result/price from PCBWay too. The biggest use for me would be to optimize order by being able to add multiple cards to same shipping/handling.
